compiler/test/dotty/tools/vulpix/ParallelTesting.scala

```diff
@@ -504,20 +504,35 @@ trait ParallelTesting extends RunnerOrchestration { self =>
       reporter
     }
 
-    private def parseErrors(errorsText: String, compilerVersion: String) =
-      val errorPattern = """.*Error: (.*\.scala):(\d+):(\d+).*""".r
-      errorsText.linesIterator.toSeq.collect {
-        case errorPattern(filePath, line, column) =>
-          val lineNum = line.toInt
-          val columnNum = column.toInt
-          val abstractFile = AbstractFile.getFile(filePath)
-          val sourceFile = SourceFile(abstractFile, Codec.UTF8)
-          val offset = sourceFile.lineToOffset(lineNum - 1) + columnNum - 1
-          val span = Spans.Span(offset)
-          val sourcePos = SourcePosition(sourceFile, span)
-
-          Diagnostic.Error(s"Compilation of $filePath with Scala $compilerVersion failed at line: $line, column: $column. Full error output:\n\n$errorsText\n", sourcePos)
-      }
+    private def parseErrors(errorsText: String, compilerVersion: String, pageWidth: Int) =
+      val errorPattern = """^.*Error: (.*\.scala):(\d+):(\d+).*""".r
+      val summaryPattern = """\d+ (?:warning|error)s? found""".r
+      val indent = "    "
+      var diagnostics = List.empty[Diagnostic.Error]
+      def barLine(start: Boolean) = s"$indent${if start then "╭" else "╰"}${"┄" * pageWidth}${if start then "╮" else "╯"}\n"
+      def errorLine(line: String) = s"$indent┆${String.format(s"%-${pageWidth}s", stripAnsi(line))}┆\n"
+      def stripAnsi(str: String): String = str.replaceAll("\u001b\\[\\d+m", "")
+      def addToLast(str: String): Unit =
+        diagnostics match
+          case head :: tail =>
+            diagnostics = Diagnostic.Error(s"${head.msg.rawMessage}$str", head.pos) :: tail
```

Comment:
If we catch warnings as well, we should handle them properly too. Although I'm not sure if we really need that and if we can handle this reliably as the external compiler process might not fail if there are some warnings but no error. As far as I know, every time we want to check if a warning was raised we enable fatal warnings and check for errors instead

Reply:
I added code to ignore warnings